application/fpga/python/DobotDriver.py

```diff
@@ -31,6 +31,8 @@
 CMD_SET_COUNTERS = 7
 CMD_GET_COUNTERS = 8
 CMD_LASER_ON = 9
+CMD_PUMP_ON = 10
+CMD_VALVE_ON = 11
```

Comment:
that's a good question I asked myself and so far I have yet to understand why there two actuators instead of one so I started by keeping them separate.